README.md

```diff
@@ -0,0 +1,163 @@
+
+### Prepare Development Environment
+
+Before you proceed, make sure you have the following software installed:
+* [Rust compiler](https://rustup.rs/) of the version that is mentioned in `rust-toolchain` file in the root of
+  [nearcore](https://github.com/nearprotocol/nearcore) project.
+* Ensure you have [AWS Credentials configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html)
+    From AWS Docs:
+
+  > For example, the files generated by the AWS CLI for a default profile configured with aws configure looks similar to the following.
+  >
+  > ~/.aws/credentials
+  > ```
+  > [default]
+  > aws_access_key_id=AKIAIOSFODNN7EXAMPLE
+  > aws_secret_access_key=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+  > ```  
+  >  
+  > ~/.aws/config
+  > ```
+  > [default]
+  > region=us-west-2
+  > output=json
+  > ```  
+
+### Compile NEAR Lake
+
+```bash
+$ cargo build --release
+```
+
+### Configure NEAR Lake
+
+To connect NEAR Lake to the specific chain you need to have necessary configs, you can generate it as follows:
+
+```bash
+$ ./target/release/near-lake --home-dir ~/.near/testnet init --chain-id testnet --download-config --download-genesis
+```
+
+The above code will download the official genesis config and generate necessary configs. You can replace `testnet` in the command above to different network ID (`betanet`, `mainnet`).
+
+**NB!** According to changes in `nearcore` config generation we don't fill all the necessary fields in the config file.
+While this issue is open https://github.com/nearprotocol/nearcore/issues/3156 you need to download config you want and replace the generated one manually.
+- [testnet config.json](https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/testnet/config.json)
+- [betanet config.json](https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/betanet/config.json)
+- [mainnet config.json](https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/mainnet/config.json)
+
+Configs for the specified network are in the `--home-dir` provided folder. We need to ensure that NEAR Lake follows
+all the necessary shards, so `"tracked_shards"` parameters in `~/.near/testnet/config.json` needs to be configured properly.
+For example, with a single shared network, you just add the shard #0 to the list:
+
+```
+...
+"tracked_shards": [0],
+...
+```
+
+### Run NEAR Lake
+
+Commands to run NEAR Lake
+
+| binary 	| Command 	| Key/Subcommand               	| Required/Default                                                 	| Responsible for                                                                                                                                                                                                                                                                                                                                                         	|
```

Comment:
I feel we can drop the `binary` column. What do you think?

Reply:
I believe it might be either confusing (for newbies) or useless (for those who understand how CLI arguments work) 🤷

Reply:
OK, agree